### PR TITLE
This PR adds an option to return ethnicity long (e.g. pop X ethnicity rather than hispanic as a seperate category)

### PR DESCRIPTION
**Pull request by dcaseykc**

### CODE REVIEW

**Comment by dcaseykc** [comment]
addresses #333

File: R/get_population.R [hunk 1]
```diff
@@ -345,27 +350,45 @@ get_population <- function(kingco = T,
...
+                             ages = ages,
+                             years = years,
+                             genders = genders,
+                             races = races,
+                             where_geo_type,
+                             where_geo_vintage,
+                             where_census_vintage,
+                             subset_by_kingco)
+    }else{
+      q = build_getpop_query(con = con,
```
[the remark this answers is not in the document]
Comment (dcaseykc) [reply R/get_population.R:376]: This is a good point. I'll remove it.

**Comment by dcolombara** [comment]
Looking good! I didn't comb through every line, but I reviewed the changes you made and ran a few of my own test. Please see my comment re: lines 376-389.

One thought I had is whether it was intentional to not provide results when race=='hispanic' and 'hispanic' %in% group_by. E.g., 
`get_pop(race_type = 'race_eth', races = 'hispanic', group_by = c('hispanic'))[]`

I would have it expected to return the same results as 
`get_pop(race_type = 'race_eth', group_by = c('hispanic'))[]` or perhaps
`get_pop(race_type = 'race_eth', group_by = c('hispanic'))[, race_eth := hispanic][]`

If you don't want to return a data.table for the example above, perhaps it should give a message explaining why?

**Comment by dcaseykc** [comment]
Does this seem sensible?

``` r
library('rads')
get_population(race_type = 'race_eth', races = 'hispanic', group_by = c('hispanic'))[]
#> Warning in get_population(race_type = "race_eth", races = "hispanic", group_by
#> = c("hispanic")): Asking for only hispanic as race grouped by hispanic does not
#> make sense. Removing hispanic from group_by
#>       pop geo_type      geo_id year   age       gender race_eth
#> 1: 259077       kc King County 2022 0-100 Female, Male Hispanic
get_population(race_type = 'race_eth', group_by = c('hispanic'))[]
#>        pop geo_type      geo_id year   age       gender     hispanic
#> 1: 2058623       kc King County 2022 0-100 Female, Male Not Hispanic
#> 2:  259077       kc King County 2022 0-100 Female, Male     Hispanic
#>                                          race_eth
#> 1: AIAN, Asian, Black, Multiple race, NHPI, White
#> 2: AIAN, Asian, Black, Multiple race, NHPI, White
get_population(race_type = 'race_eth', group_by = c('hispanic'))[, race_eth := hispanic][]
#>        pop geo_type      geo_id year   age       gender     hispanic
#> 1: 2058623       kc King County 2022 0-100 Female, Male Not Hispanic
#> 2:  259077       kc King County 2022 0-100 Female, Male     Hispanic
#>        race_eth
#> 1: Not Hispanic
#> 2:     Hispanic
```

**Comment by dcolombara** [comment]
Your warning message makes sense to me. 
I'm not accepting this pull request yet because (a) I thought you wanted to add some stuff to the wiki and (b) your request is to pull into main rather than dev. 
If this is the final tweak, and you want a solid review, I can do it on Monday. Otherwise, I think you're ready to move forward.

**Comment by dcaseykc** [comment]
I'll add to the wiki next week, run tests, and pull into dev if it passes.

**Comment by dcaseykc** [comment]
@dcolombara: Tests pass and get population vignette has been updated. I'll regenerate the wiki when this eventually comes into main. Would you mind doing a quick once over and then I'll merge it?

**Review by dcolombara — COMMENTED** [review]
Looks really good. I updated the help file and the vignette in an attempt to make somethings more clear. I also tweaked the tests. The the function is untouched. 

The help file still has one thing that didn't make sense to me, but I didn't want to delete it because it might have been my edits that made it no longer make sense. Under the `group_by` description, what does the following mean? <It can be combined with "race_eth".> 

If my changes look good to you, feel free to pull into dev. Even if they don't and you want to revert to some of the older documentation, I still think you can pull into dev.

Once this is pulled into dev, can we delete the branch?

**Pull request merged**